.github/workflows/test.yaml

```diff
@@ -82,6 +82,7 @@ jobs:
           run: pytest -m "not fuzzing" -s --cov=src -n auto
           env:
             WEB3_INFURA_PROJECT_ID: ${{ secrets.WEB3_INFURA_PROJECT_ID }}
```

Comment:
Could we support multiple keys in just the `WEB3_INFURA_PROJECT_ID` by splitting "," always? If only 1, itll be a list of just 1.

Reply:
I like this suggestion, and would be backwards-compatible

Reply:
Sounds better. I will make the change. This way we don't have to add extra env vars